client/zh_cn.editable.json

```diff
@@ -417,19 +417,19 @@
     "projectelo.tip_message.31": "像猪灵和凋零骷髅这样的生物会尽量躲开熔岩、火和岩浆块。",
     "projectelo.tip_message.32": "绯红和诡异森林的木头可以合成几乎所有木头能合成的工具... 除了船。",
     "projectelo.tip_message.33": "在堡垒遗迹中桥部分的磁石箱子里不会含有黑曜石，但是可以含有铁锭。",
-    "projectelo.tip_message.34": "即使看不见打开箱子的玩家，猪灵仍然会愤怒 1-2 秒。",
-    "projectelo.tip_message.35": "制作光灵箭需要很多萤石，但可以将你的箭的数量乘2。",
+    "projectelo.tip_message.34": "即使失去了打开箱子玩家的视野，猪灵仍然会愤怒 1-2 秒。",
+    "projectelo.tip_message.35": "制作光灵箭需要很多萤石，但可以将您的箭的数量翻倍。",
     "projectelo.tip_message.36": "恶魂火球的爆炸威力为 1，TNT 为 4，而床和重生锚的爆炸威力为 5。",
     "projectelo.tip_message.37": "对着一个已经被填充的末地传送门框架上右键末影之眼，它也不会被丢出去。",
     "projectelo.tip_message.38": "每个末地传送门框架在生成时有 10% 的概率含有末影之眼。",
     "projectelo.tip_message.39": "传送门房间只能在离要塞初始楼梯的至少 7 个房间外生成。",
-    "projectelo.tip_message.40": "传送门房间不像其它要塞房间一样，它并不会被自然生成的地形 (比如峡谷) 所覆盖。",
-    "projectelo.tip_message.41": "每个末影岛将生成相同的 10 根黑曜石柱，只是布局不同。",
+    "projectelo.tip_message.40": "传送门房间不像其它要塞房间一样会被自然生成的地形 (比如峡谷) 所覆盖。",
+    "projectelo.tip_message.41": "每个末影岛将生成相同的 10 根同样的黑曜石柱，只是布局不同。",
```

Comment:
“末地主岛”maybe better than“末影岛”？

Reply:
and i think there are still some controversies, like“排名分","排名赛模式”, instead of"排位分","排位模式"

Reply:
and ”大厅“ , ”房间“ i'd say most of players call a room”房间“ instead of ”大厅“

Reply:
sorry,but these translation type is I learn from my other friends,i will change the "末影岛" to more accurate "末影主岛",but I will keep other translation like "Lobby/大厅" and "Ranked/排名赛" for a while,all translation changes will depends on player feedback,thank you

Reply:
yeah i respect that, but still, it's ”末地主岛“ not "末影" . And as far as i can see, there's "room" in ranked and i didn't see any "lobby", maybe i'm wrong here but... i'm not sure.

Reply:
fixed the typo,my bad,but the room or lobby,I do prefer lobby a little more,as far as I know and some streamer record,they all call it Party or Lobby,also the en_us file use the word that close to Lobby or Party too,I'll wait for your feedback,if more players prefers to Room/房间,I will change it after you reply,if I'm not busy

Reply:
yeah i compared yours with en_us file in DO_NOT_CHANGE directory.(i didn't know if it's outdated), and all there are just "room"... so which en_us file did you follow?

Reply:
committed ;)

Reply:
I'm glad we reached a consensus  :3

Reply:
in fact I use Lobby more is I saw from other players,at least to me is:join my lobby,at some point this are right ya'know,as a game party owner it doesn't have any problem to call it a "lobby",but it depends on player base,Chinese Community is apparently larger than others

Reply:
> in fact I use Lobby more is I saw from other players,at least to me is:join my lobby,at some point this are right ya'know,as a game party owner it doesn't have any problem to call it a "lobby",but it depends on player base,Chinese Community is apparently larger than others

yeah this is one of fun things in translation, you use a word in English but at the same situation, you don't use the literal translation when using another language, because there are already suitable words to use there.

Reply:
I wonder what kind of communicate software use most in China,see maybe I can make one

Reply:
Tencent qq mostly

Reply:
see if I can create one,maybe communicate will much easier in future

Reply:
3656487283 thank you,no region lock